### DISCUSSION TITLE
How to enable desktop support ?

### DISCUSSION THREAD

**Discussion by TeddyLourson** [isar/isar#101] [Help]
Hi 👋 !
I want to use Isar for my Flutter app.
Why ? Because it's a new project and I don't want to switch from Hive to Isar later on.
But I'm having an error trying to open isar with `await openIsar()` using macOS.

`IsarError (IsarError: Could not initialize IsarCore library. If you create a Flutter app, make sure to add isar_flutter_libs to your dependencies. Isar does not support 32-bit processors so make sure that your device / emulator has a 64-bit processor.)`

I do have `isar_flutter_libs` in my `pubspec.yaml`

I saw that I have to add binaries for each platforms (in my case, Linux, Windows and MacOS) but I have no idea how...
I would really appreciate some explanations on how to use Isar with desktop.

Thank you very much for your packages !
I loved Hive and I really want to use this one 🤞 !

**Comment by TeddyLourson** [comment]
I've switched to Hive, everything's working fine, I'll migrate to Isar when it's ready for Desktop.

**Comment by michalisioak** [comment]
this issue might be helpfull 
#12

**Reply by iota9star** [reply]
> this issue might be helpfull
> #12

I don't know where to put .dll file? Can you tell me where the. DLL is?

**Reply by TeddyLourson** [reply]
I knew about the libs but I don't know where to put them.
Plus it's said to be very unstable so I decided to go on with hive instead and I'll switch later on when desktop support for Isar will be stable. It won't be that much of an issue since I layered my application's code. It's just some more work to put on top of everything else.

**Reply by iota9star** [reply]
download [the latest binary](https://github.com/isar/isar-core/releases) and rename to `libisar.dll`, copy it to `build/windows/runner/Debug` or `build/windows/runner/Release`, which will work fine.

**Reply by TeddyLourson** [reply]
Oh ! There are other versions of the binaries ?!
I’ll give it a try then !
I’m on MacOS but I guess it should be a pretty similar location.
Thanks !

**Reply by iota9star** [reply]
the repo provides some clue for macos. 

![image](https://user-images.githubusercontent.com/21272154/117685597-a729de00-b1e8-11eb-9100-c26cbf578b1f.png) 
![image](https://user-images.githubusercontent.com/21272154/117685656-b6a92700-b1e8-11eb-9cc8-df70d3121004.png)
![image](https://user-images.githubusercontent.com/21272154/117685842-e22c1180-b1e8-11eb-8e14-7244e08cd317.png)
